app/connector/fhir/src/main/java/io/syndesis/connector/fhir/FhirMetadataRetrieval.java

```diff
@@ -199,7 +198,7 @@ private String newResourceSpecification(String type, String containedResourceTyp
             specification = Resources.getResourceAsText("META-INF/syndesis/schemas/dstu3/" + resourcePath + ".json", FhirMetadataRetrieval.class.getClassLoader());
 
             if (ObjectHelper.isNotEmpty(containedResourceTypes)) {
-                String[] containedResourceTypesSplit = StringUtils.split(containedResourceTypes, ConfigurationProperty.MULTIPLE_SEPARATOR);
+                String[] containedResourceTypesSplit = Json.reader().forType(String[].class).readValue(containedResourceTypes);
```

Comment:
Formik posts the value for multi select as a string e.g. '["Account", "Patient"]' and not as a json array ["Account", "Patient"]. Not sure, if it's a feature or bug, but changed the backend accordingly.